src/mitsubishi2mqtt/config.h

```diff
@@ -102,6 +102,7 @@ String temp_step                   = "1"; // Temperature setting step, check val
 
 // sketch settings
 const PROGMEM uint32_t SEND_ROOM_TEMP_INTERVAL_MS = 30000; // 45 seconds (anything less may cause bouncing)
```

Comment:
@gysmo38 I noticed this was changed from 45 seconds (like the comment says) to 30 seconds in 2f439c64. Do you recall why? What does "bouncing" mean?

Reply:
I try to understand why my unit is not very responsive. I put 30000, it look better but I have other issue. I will try to get back at 45000.
Can I merge this PR?

Reply:
Yes. I've been running this code on my heat pumps for about a month now.